src/pages/Dashboard/Dashboard.js

```diff
@@ -0,0 +1,151 @@
+import React from "react";
+import TinderCard from "react-tinder-card";
+
+import { useRef, useMemo, useEffect } from "react";
+import { useState } from "react";
+import "./Dashboard.css";
+import Matches from "../../components/Matches/Matches";
+
+const db = [
+  {
+    name: "Ojasvi",
+    age: 21,
+    url: "https://www.scrolldroll.com/wp-content/uploads/2022/12/Hania-Amir-beautiful-pakistani-actresses.jpg",
+  },
+  {
+    name: "Ira",
+    age: 22,
+    url: "https://imgeng.jagran.com/webstories/7791/pakistani-actress-hania-amir-is-breathtaking-in-these-western-looks-1671693359.jpeg",
+  },
+  {
+    name: "Aanya",
+    age: 23,
+    url: "https://cdn.siasat.com/wp-content/uploads/2023/02/hania-amir.jpg",
+  },
+  {
+    name: "Lasya",
+    age: 24,
+    url: "https://lollywoodcity.com/wp-content/uploads/2022/03/haniaheheofficial_272875010_971176673531365_4582770850327121465_n-819x1024.jpg",
+  },
+  {
+    name: "Aakriti",
+    age: 25,
+    url: "https://imgeng.jagran.com/webstories/7791/pakistani-actress-hania-amir-is-breathtaking-in-these-western-looks-1671693359.jpeg",
+  },
+];
+
+const Dashboard = () => {
+  const characters = db;
+  const [lastDirection, setLastDirection] = useState();
+  const [currentIndex, setCurrentIndex] = useState(db.length - 1);
+  const currentIndexRef = useRef(currentIndex);
+  const childRefs = useMemo(
+    () =>
+      Array(db.length)
+        .fill(0)
+        .map((i) => React.createRef()),
+    []
+  );
+  const updateCurrentIndex = (val) => {
+    setCurrentIndex(val);
+    currentIndexRef.current = val;
+  };
+  const canGoBack = currentIndex < db.length - 1;
+
+  const canSwipe = currentIndex >= 0;
+  const swiped = (direction, nameToDelete, index) => {
+    setLastDirection(direction);
+    updateCurrentIndex(index - 1);
+  };
+
+  const outOfFrame = (name, idx) => {
+    console.log(`${name} (${idx}) left the screen!`, currentIndexRef.current);
```

Comment:
In General it is advised to remove all the console before merging the changes.

Reply:
- Removed console.log from code